pkg/util/sys/device.go

```diff
@@ -322,3 +326,31 @@ func parseKeyValuePairString(propsRaw string) map[string]string {
 
 	return propMap
 }
+
+// FindRBDMappedFile search for the mapped RBD volume and returns its device path
+func FindRBDMappedFile(imageName, poolName, sysBusDir string) (string, error) {
```

Comment:
having something RBD specific in the `sys` package feels a little odd.  is there a better place for it?

Reply:
yeah i hear you. I tried putting it in a different place but i got all kinds of import cycle errors. I could create a new package called 'rbdutils' and have it there. What do you think?

Reply:
i don't have a very strong feeling about it, so you can leave it as is.

Reply:
yeah this does not belong here.

Reply:
ok i will create a `rbdutils` and add it there.